RATIONALE.md

```diff
@@ -65,4 +65,102 @@ KQueen supplies the backend API for provider-agnostic cluster management. It ena
 * **Update** - install newer version of Kubernetes
 * **Autoscale** - watch Kubernetes scheduler or pods and start new minions when all existing minions are fully utilized
 * **Manage addons** - enable or disable cluster addons
-
+* **Members managing** - Its possible to define own mail server or use default to provide user registering into Kqueen.
+Configure mail service in docker-compose.production.yml:
+```
+  mail:
+    image: modularitycontainers/postfix
+    volumes:
+      - /var/spool/postfix:/var/spool/postfix
+      - /var/mail:/var/spool/mail
+    environment:
+      MYHOSTNAME: 'mail'
+
+ ```
+Configure following variables in kqueen ui service:
+```
+  KQUEENUI_MAIL_SERVER: mail
+  KQUEENUI_MAIL_PORT: 10025
+```
+Pay attention that volume-mapping for mail containers is additional feature. It used for store mailing history and forward additional postfix mail configuration (it should configured properly on local machine),
+otherwise its possible to run mail server without volume-mapping. After all configuration steps, user should invite members using email notification. Member should receive mail with activation link to Kqueen service and possibility of password-setting.
+Superadmin also can manage member roles.
+* **Metrics collecting** - Its possible to add external Prometheus serveri to extend monitoring in Kqueen. In case of using external server, need to include rules from kqueen/prod/prometheus into existing Prometheus service. Otherwise, its possible to use predefined Prometheus, defined in docker-compose.production.yml.
+Configure prometheus service (ip address/port, volumes) in docker-compose.production.yml. Define  scraper IP address for PROMETHEUS_WHITELIST variable in kqueen api service section.
+All application metrics exported on **/metrics** and this metric can scraped by any external Prometheus instance. There two ways to authenticate to get this endpoint:
+
+- Kqueen API
+```
+TOKEN=$(curl -s -H "Content-Type: application/json" --data '{"username":"admin","password":"default"}' -X POST localhost:5000/api/v1/auth | jq -r '.access_token'); echo $TOKEN
+curl -H "Authorization: Bearer $TOKEN" {kqueen_api_host}:5000/metrics
+```
+
+
+- Prometheus API
+Add scraper IP address to PROMETHEUS_WHITELIST configuration and then:
+```
+curl {prometheus_host}:{prometheus_port}/metrics
+```
+
+### Provisioner configuration settings
+
+* **Google Kubernetes Engine**
+
+# Register account on GCE (https://console.cloud.google.com)
```

Comment:
All your # is a first-level headers, did you want to * instead?
Also I suggest to remove * from * **Google Kubernetes Engine** and add to the list of provided steps, smth like:
## Provisioner configuration settings ## -2d level section

### **Google Kubernetes Engine** ### -3d level section
* Register

Reply:
- one
- two

Reply:
*. one
*.two

Reply:
#. one
#. two

Reply:
1. one
2. two